tensorflow/core/kernels/mkl_fused_ops_test.cc

```diff
@@ -242,7 +242,7 @@ class MklFusedConv2DOpTest : public OpsTestBase {
     if (std::find(fused_ops.begin(), fused_ops.end(), "Elu") !=
         fused_ops.end()) {
       last_op = "with_elu";
-      next_op = ops::Elu(root.WithOpName(last_op), next_op);
+      next_op = ops::Relu(root.WithOpName(last_op), next_op);
```

Comment:
Could you please add a TODO in the code, saying that this is intentional and will be fixed later? Please use `TODO(<your_username>):` or `TODO(intel-tf):`. Thank you!

Reply:
There're 2 Elu issues, I've fixed one related MatMul in this PR, and added comment for another related to Conv. Please take a look, thanks!